src/components/structures/RoomView.js

```diff
@@ -748,6 +754,21 @@ export default createReactClass({
             }
         }
     },
+    onEventDecrypted(ev) {
+    if (!SettingsStore.getValue('dontShowChatEffects')) {
+        if (ev.isBeingDecrypted() || ev.isDecryptionFailure() ||
+            this.state.room.getUnreadNotificationCount() === 0) return;
```

Comment:
I don't think this will do what you want.
I suggest doing something similar to Notifier, maybe even adding something generic for both Notifier and this to use.

Notifier does something like such:
Before the Matrix client is ready (has done initial sync) ignore all events.
`onEvent`: Evaluate all non-encrypted events, add encrypted event IDs to a Set.
`onEventDecrypted`: If event ID is in the Set, remove it from Set and evaluate it. Otherwise ignore it.